src/to_geo.rs

```diff
@@ -18,11 +19,7 @@ impl<'a, 'b$(,$lt)?> TryFrom<&'b $ty_name<'a$(,$lt)?>> for Geometry<f64> {
         let wkt_obj = wkt::Wkt::from_str(&wkt_str)
```

Comment:
```suggestion
        geo_types::Geometry::try_from_wkt_str(&wkt_str)
```

With wkt 0.10+, you should be able to do this instead, without ever interacting directly with the `Wkt` struct.

Reply:
I'm not sure if it's relevant to this crate, but we also added support for converting directly to the inner enum member:
```rust
geo_types::Point::try_from_wkt_str(&wkt_str)
geo_types::LineString::try_from_wkt_str(&wkt_str)
// etc.
```

Reply:
Thanks for the suggested improvement! As for the second point, I'm not sure if it's supposed to be handled in here or not...

Reply:
Thank you! I went ahead and made the suggested change. I also simplified the unit test with more reasonable coordinates as opposed to the exact ones that failed in my system when I noticed the bug.